src/js/firstDiscoveryEditor.js

```diff
@@ -209,14 +209,31 @@ https://github.com/fluid-project/first-discovery/raw/master/LICENSE.txt
     };
 
     /*
-     * To integrate the first discovery tool with the preferences server
+     * To integrate the first discovery tool with the preferences server.
+     * This component should be used as an add-on component for gpii.firstDiscovery.firstDiscoveryEditor
```

Comment:
Do you mean to say "add-on grade" instead of "add-on component" or did you mean subcomponent?